script/gen_examples.sh

```diff
@@ -0,0 +1,20 @@
+set -ex
+
+# Copies the dev examples into the `examples` root and replaces the local 
+# fgb libary with the published hosted one
```

Comment:
This script might be controversial... maybe you'll have a better suggestion.

The problem I'm trying to solve is that I want to be able to have a "local" version of the examples that I can dev on and which use the locally built FGB libraries.

...but I don't want to have to maintain twice as many example HTML files just for this small change — hence this script. It's admittedly pretty dumb. I am open to alternatives, or if you find it grossly offensive, I could just get rid of the local-dev versions and keep editing them each time I want to work locally.

Though a side benefit is, next time you release, you don't have to update all the examples to point to the latest release on unpkg.com, just this script. 😉